leaphymicropython/utils/i2c_helper.py

```diff
@@ -1,6 +1,48 @@
+"""This module provides helper functions for i2c."""  # Single-line docstring
+
 import struct
 
 
+def i2c_is_it_used_and_alive(func):
+    """check if i2c connection is used and if so, if it is alive before using it"""
+
+    def wrapper(*args, **kwargs):
+        instance = args[0]
+        # check if i2c is used
+        i2c_used = hasattr(instance, "i2c")
```

Comment:
Dit kunnen we mooier oplossen met een abstracte "interface" class en een issubclass() check. Daar kunnen dan ook de benodigde methods in zoals initialize() etc.